corehq/apps/users/templates/users/partials/bootstrap5/edit_role_modal.html

```diff
@@ -144,33 +144,33 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
             </legend>
 
             <div data-bind="foreach: reports">
-              <div class="form-group" data-bind="visible: visibilityRestraint">  {# todo B5: css-form-group #}
-                <label class="col-md-4 form-label" data-bind="text: text"></label>
+              <div class="row" data-bind="visible: visibilityRestraint">
+                <label class="col-md-4 form-label fw-bold" data-bind="text: text"></label>
                 <div class="col-md-8 controls">
-                  <div class="form-check">
-                    <input type="checkbox"  {# todo B5: css-checkbox #}
+                  <div class="form-check pt-0">
+                    <input class="form-check-input" type="checkbox"
                            data-bind="attr:{id: checkboxLabel}, checked: checkboxPermission" />
-                    <label data-bind="attr:{for: checkboxLabel}, text: checkboxText">
+                    <label class="form-check-label" data-bind="attr:{for: checkboxLabel}, text: checkboxText">
                     </label>
                   </div>
                 </div>
               </div>
             </div>
-            <div class="form-group" data-bind="visible: !reportPermissions.all()">  {# todo B5: css-form-group #}
-              <label class="col-md-4 form-label">
+            <div class="row" data-bind="visible: !reportPermissions.all()">
+              <label class="col-md-4 form-label fw-bold">
```

Comment:
I don't think this should be a label because it doesn't have an associated input. Maybe a header if it's describing what comes next?